src/BlockTypes/Checkout.php

```diff
@@ -401,17 +401,25 @@ public static function include_token_id_with_payment_methods( $list_item, $token
 	protected function register_block_type_assets() {
 		parent::register_block_type_assets();
 		$blocks = [
-			'checkout-blocks/express-payment',
+			'checkout-blocks/actions',
+			'checkout-blocks/billing-address',
 			'checkout-blocks/contact-information',
+			'checkout-blocks/express-payment',
+			'checkout-blocks/fields',
+			'checkout-blocks/order-note',
+			'checkout-blocks/order-summary',
+			'checkout-blocks/order-summary-cart-items',
+			'checkout-blocks/order-summary-coupon-form',
+			'checkout-blocks/order-summary-discount',
+			'checkout-blocks/order-summary-fee',
+			'checkout-blocks/order-summary-shipping',
+			'checkout-blocks/order-summary-subtotal',
+			'checkout-blocks/order-summary-taxes',
+			'checkout-blocks/payment',
 			'checkout-blocks/shipping-address',
-			'checkout-blocks/billing-address--checkout-blocks/shipping-address',
-			'checkout-blocks/billing-address',
 			'checkout-blocks/shipping-methods',
-			'checkout-blocks/payment',
-			'checkout-blocks/order-note',
-			'checkout-blocks/actions',
 			'checkout-blocks/terms',
-			'checkout-blocks/order-summary',
+			'checkout-blocks/totals',
 		];
```

Comment:
I attempted to write a code that traverse the `checkout-blocks` folder to get all files, but my code felt very hacky :/

```php
foreach ( new \DirectoryIterator( dirname( dirname( __DIR__ ) ) . '/build/checkout-blocks' ) as $file ) {
			if ( $file->isFile() ) {
					print $file->getFilename() . "\n";
			}
		}
```

Maybe @dinhtungdu or @wavvves you know a better way without me hardcoding the build file and needing to traverse folders manually up.

Reply:
> needing to traverse folders manually up.

We can use the `Package::get_path()` to avoid traversing folders:

```
foreach ( \Automattic\WooCommerce\Blocks\Package::get_path() . 'build/checkout-blocks' ) as $file ) {
```

Reply:
I think if we only need file paths, we can use `glob()` instead of `DirectoryIterator`.

`glob()` consumes less memory because it doesn't create any object, but the difference is negligible IMO.